### PR TITLE
FAD-4414 Enable Save Results Buttons in Tools UI

### PR DESCRIPTION
**Pull request by jonambas**

### CODE REVIEW

**Comment by colestrode** [comment]
Not really in this review, but I noticed the "SparkPost Dashboard" opens in the same window, it might be nice to open the SparkPost app in a separate window.

File: src/pages/dkim/components/ResultDetailHeader.js [hunk 1]
```diff
@@ -1,52 +1,36 @@
...
-          </div>
-          {this.renderDetails()}
-        </div>
+const ResultDetailRow = ({ rows }) => (
+  <span>
+    {rows.map((row, key) => (
+      <div className='dkimResultDetailHeader' key={key}>
+        {row.map((value, key) => {
+          if (key === 0) {
+            return <span className='dkimResultDetailHeader__label' key={key}>{value}</span>;
```
Comment (colestrode) [inline src/pages/dkim/components/ResultDetailHeader.js]: could you do this with `classnames`? that may be a little cleaner

File: src/components/button/Button.js [hunk 2]
```diff
@@ -45,41 +46,46 @@ const LinkButton = (props) => {
...
-  const { to = null, onClick = null, title = '', children } = props;
+  const { to = null, external = null, onClick = null, title = '', children } = props;
 
-  return (
-    <Link to={to} onClick={onClick} className='actionLink' title={title}>
-      {children}
-    </Link>
-  );
+  if (external) {
+    return <a href={external} onClick={onClick} className='actionLink' title={title}>{children}</a>;
```
Comment (colestrode) [inline src/components/button/Button.js]: Ah... I added this a general comment, but it looks like this is where we'll need to add a `target=_blank` to open these external links in a new window.

Comment (jonambas) [inline src/components/button/Button.js]: Would the save results button also open a new tab? or just the login/sign up links in the nav?

**Comment by colestrode** [comment]
Looks like you have conflicts to resolve, but once those are clear :shipit:!

**Pull request merged**